app/javascript/src/ui/common/AsyncSelect.tsx

```diff
@@ -1,21 +1,29 @@
 import * as React from "react";
 import debounce from "lodash-es/debounce";
-import AsyncSelect, { Props as AsyncProps } from 'react-select/lib/Async';
-import Createable from "react-select/lib/Creatable";
+import { ActionMeta, ValueType } from "react-select/lib/types";
+import AsyncSelect, { Props as AsyncProps } from "react-select/lib/Async";
+import Createable, { Props as CreateableProps } from "react-select/lib/Creatable";
+import AsyncCreatable from "react-select/lib/AsyncCreatable";
+
 import { formDialogClass } from "ui/common/FormModal";
 import { SelectOption } from "ui/common/RenewalForm";
 import Form from "ui/common/Form";
-import { ActionMeta } from "react-select/lib/types";
 
-interface Props extends AsyncProps<any> {
-  name: string;
-  createable?: boolean;
+interface Props {
+  name?: string;
```

Comment:
Why make name optional? I had this as required as a lot of the Form magic needs it.

Reply:
I only made it optional because `getFormRef` is optional. If `getFormRef` is not defined and you use this as a standalone component (not within a form), then `name` is not necessary.

That being said, this change isn't really relevant to this PR any more anyway, and there wasn't a particularly good reason for it. I can revert.

Reply:
okay makes sense. You can leave it as is